src/components/LegalMentions/LegalMentions.js

```diff
@@ -1,27 +1,76 @@
-import styled from 'styled-components'
-import theme from '../../styles/theme'
+import React from 'react'
 
-const LegalMentionsStyled = styled.div`
-.legal-mentions {
-    margin-top: ${theme.spacing.XXXL};
-    margin-bottom: ${theme.spacing.XXXL};
-    font-size: ${theme.spacing.S};
-    color: ${theme.color.logoGrey};
+import Header from '../Header/Header'
 
-    p a, strong {
-        color: ${theme.color.white};
-    }
+import LegalMentionsStyled from './LegalMentions.styled' 
 
-    &-title {
-        font-size: ${theme.font.XL};
-        margin-bottom: ${theme.spacing.M};
-        color: ${theme.color.white};
+const LegalMentions = () => (
```

Comment:
maybe we can put the LegalMentions in markdown to it's easier to read and split them for intertionalization. 
Could be done in another PR though.